### DISCUSSION TITLE
Multiple content-scripts support

### DISCUSSION THREAD

**Discussion by michaldolensky** [antfu/vitesse-webext#31] [Q&A]
Hello,
What's best way to add more entries to content scripts and make them independent from each other? 
I would like to load dynamically them from the background script based on defined URLs and toggle options. 

Thank you.

**Comment by tshemeng** [comment]
At runtime, into specific tabs or urls, using the `tabs.executeScript()` API, read in the following article

- MDN [Loading content scripts](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#loading_content_scripts)
- Google Developer [Inject programmatically](https://developer.chrome.com/docs/extensions/mv3/content_scripts/#programmatic)

Note that Firefox requires file path to start with `/` and Chromium-based browsers require `./` to begin with
